about.html

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="about.css">
+  <link href='https://fonts.googleapis.com/css?family=Lato:400,700' rel='stylesheet' type='text/css'>
+
+  <title>About</title>
+</head>
+<body>
+  <header>
+    <nav>
+    <div class="small-navbar desktop">
+      <a href="#"><img src="images/facebook.png" alt=""></a>
+      <a href="#"><img src="images/twitter.png" alt=""></a>
+      <p>English</p>
```

Comment:
- [ ] As [suggested by the previous reviewer](https://github.com/danielf-canon/Capstone-project/pull/1#discussion_r1289245573), please make sure to include the appropriate `alt` attribute for images. The `alt` attribute provides accessibility for visually impaired users, conveys image context, aids SEO, and offers fallback content when images fail to load 💪

  _This applies to the `index.html` as well_